doc/sphinxext/numpydoc/compiler_unparse.py

```diff
@@ -0,0 +1,865 @@
+""" Turn compiler.ast structures back into executable python code.
```

Comment:
Not really super important, but for the numpydoc stuff you may want to take it from what is done in joblib. There are some files that are not used and removed in numpydoc master.